microprofile-lra/src/main/java/org/wildfly/quickstarts/microprofile/lra/JaxRsApplication.java

```diff
@@ -25,6 +25,6 @@
 import jakarta.ws.rs.ApplicationPath;
 import jakarta.ws.rs.core.Application;
 
-@ApplicationPath("/")
+@ApplicationPath("/rest")
```

Comment:
`/` needs to be reserved for the root web page as required. Changed to `/rest` because there can be path conflicts if we don't change the rest endpoints path to something else other than `/`